pkg/gpu/nvidia/health_check/health_checker.go

```diff
@@ -159,6 +166,56 @@ func (hc *GPUHealthChecker) addMigEnabledDevice(deviceName string, device *nvml.
 	return nil
 }
 
+type callDevice interface {
+	parseMigDeviceUUID(UUID string) (string, uint, uint, error)
+}
+type GPUDevice struct{}
+
+func (gd *GPUDevice) parseMigDeviceUUID(UUID string) (string, uint, uint, error) {
+	return nvml.ParseMigDeviceUUID(UUID)
+}
+
+func (hc *GPUHealthChecker) catchError(e nvml.Event, cd callDevice) {
+	if e.Etype != nvml.XidCriticalError {
+		return
+	}
+	// Only marking device unhealthy on Double Bit ECC Error
```

Comment:
Shall we change this comment? As after this change, customized error and Double Bit ECC Error (48) will all be checked.